lib/app.rb

```diff
@@ -1,10 +1,12 @@
 require_relative 'board'
+require_relative 'move_requester'
 
 # A class to manage the running of a game
```

Comment:
I know it isn't part of this pull request, but I'm not sure if the comment brings much to the file. If you think you need it to clarify what the class `App` does, then it might be a sign that you need to rename the class to something like `Game` which might make it more self explanatory, removing the need for the comment?